### DISCUSSION TITLE
debug error

### DISCUSSION THREAD

**Discussion by keyvanplays** [CosmosOS/Cosmos#2964] [Q&A]
i have this error when i run
![Screenshot 2024-02-26 053851](https://github.com/CosmosOS/Cosmos/assets/69101158/1e8e00e2-628a-447a-b618-f674ca2467ab)

**Comment by randomusert** [comment]
[Are you building iso file](https://github.com/CosmosOS/Cosmos/issues/2660)

**Comment by Bokobelin** [comment]
I thiink you are trying to use Qemu when it wants you to use VMWarePlayer/Workstation.

**Reply by randomusert** [reply]
Yeah i very well Be it because i use VMware and it Works fine For debugging

**Reply by ghost** [reply]
> Yeah i very well Be it because i use VMware and it Works fine For debugging

True Works Fine For Me But I Usually Don't Debug As The Final Build Will Be Without Debug Also I Am Too Good For Debuging
I Have Used The Debugger In The Past I Don't Any More

**Comment by randomusert** [comment]
Try to disable debugging and If that doesn't Work just open a issue i quess
